interface/usergroup/addrbook_edit.php

```diff
@@ -87,7 +87,7 @@ function invalue($name) {
   // 2 = Person Centric
   // 3 = Company Centric
   $sql = sqlStatement("SELECT option_id, option_value FROM list_options WHERE " .
-   "list_id = 'abook_type'");
+   "list_id = 'abook_type' AND activity = 1");
```

Comment:
Does this mean Active users are the only thing that can be selected here?  or does activity=1 mean something else?